reader/components/outline-sidebar/OutlineSidebar.jsx

```diff
@@ -46,11 +46,20 @@ class OutlineSidebar extends React.PureComponent {
     ]
 
     return (
-      <OutlineGroup
-        isExpanded
-        linkService={linkService}
-        outline={finalOutline}
-      />
+      <>
+        {!outline && (
```

Comment:
Could not it be an empty array? If so, you should also check for a `length`

Reply:
This is something I don't know. I will add it there just to be sure.

Reply:
https://github.com/oacore/reader/pull/106/commits/a8d706477d649c2151095655eeaaded63666f0a7